KachingPlugIn/Factories/ProductFactory.cs

```diff
@@ -148,12 +158,14 @@ public Product BuildKaChingProduct(ProductContent product, IList<string> tags, s
                 /* Find an image */
                 /* ---------------------------- */
 
-                var media = variation.CommerceMediaCollection.FirstOrDefault();
-                if (media != null)
+                CommerceMedia variantImage = product.CommerceMediaCollection.FirstOrDefault();
+                if (productImage != null)
```

Comment:
It looks like it should be `variation.CommerceMediaCollection.FirstOrDefault();` here to get the variant image? Also it looks like the check against null should be against `variantImage`

Reply:
Fixed now.